modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/DLPortlet.java

```diff
@@ -41,8 +41,6 @@
 		"com.liferay.portlet.css-class-wrapper=portlet-document-library",
 		"com.liferay.portlet.display-category=category.cms",
 		"com.liferay.portlet.display-category=category.highlighted",
-		"com.liferay.portlet.footer-portlet-javascript=/document_library/js/legacy/main.js",
```

Comment:
<img src="https://media2.giphy.com/media/M90mJvfWfd5mbUuULX/giphy.gif"/>



The messages were annoying!